frontends/web/src/locales/en/app.json

```diff
@@ -24,7 +24,17 @@
     "extendedPublicKey": "Extended public key",
     "label": "Account info",
     "title": "Account information",
-    "verify": "Verify on device"
+    "verify": "Verify on device",
+    "xpubTypeChangeBtn": {
+      "p2pkh": "View legacy P2PKH extended public key",
+      "p2wpkh": "Switch back to Native segwit (bech32)",
+      "p2wpkh-p2sh": "View older segwit extended public key"
+    },
+    "xpubTypeInfo": {
+      "p2pkh": "Currently displaying legacy P2PKH extended public key ({{current}} of {{numberOfXPubs}})",
```

Comment:
what if you used only one translation `Currently displaying {{scriptType}} extended public key ({{current}} of {{numberOfXPubs}})`?

This would avoid many translations, and also we don't need to come up with half-accurate names for the script types (e.g. 'segwit extended public key' is technically also true for taproot, as taproot is also segwit, etc).

Reply:
then adding another type like taproot would work automatically too.

Reply:
yes and I'll move the more human readable name to utils and use it here ok?

```diff
diff --git a/frontends/web/src/routes/account/utils.ts b/frontends/web/src/routes/account/utils.ts
index 515b7732..0b409441 100644
--- a/frontends/web/src/routes/account/utils.ts
+++ b/frontends/web/src/routes/account/utils.ts
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-import { CoinCode } from '../../api/account';
+import { CoinCode, ScriptType } from '../../api/account';
 
 export function isBitcoin(code: string): boolean {
     switch (code) {
@@ -63,3 +63,15 @@ export function getCoinCode(coinCode: CoinCode): CoinCode | undefined {
             return 'eth';
     }
 }
+
+
+export function getScriptTypeName(scriptType: ScriptType): string {
+    switch (scriptType) {
+        case 'p2pkh':
+            return 'Legacy';
+        case 'p2wpkh-p2sh':
+            return 'Segwit';
+        case 'p2wpkh':
+            return 'Native segwit (bech32)';
+    }
+}
```

Reply:
okay but drop the (bech32) maybe? Afaik this word is not explained anywhere.

Reply:
Good point, I'll change back, but keep the script type name function it in utils.